k8s/templates/router.yaml

```diff
@@ -83,7 +83,7 @@ spec:
           containerPort: 80
         env:
         - name: force_update
-          value: "3"
+          value: "1"
```

Comment:
what would `3` mean?

Reply:
The `force_update` in here is old. I did not have the time yet to make this nicer. Basically when this changes it forces a redeploy of the internal router if only the config changed. I'd prefer an automatic way. But since it changes rarely that a good enough workaround.